go.mod

```diff
@@ -81,3 +81,5 @@ require (
 	launchpad.net/gocheck v0.0.0-20140225173054-000000000087 // indirect
 	layeh.com/gopher-json v0.0.0-20180103211521-1aab82196e3b
 )
+
+go 1.13
```

Comment:
please rebase on master, it's already there https://github.com/zalando/skipper/blob/master/go.mod#L49